### PR TITLE
Update remove-former-employee-step-4.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated instructions to get access to onedrive

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6484

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/add-users/remove-former-employee-step-4.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ To preserve a former user's OneDrive files, first give yourself access to their
 
 2. Select a user.
 
-3. In the right pane, select **OneDrive**. Under **Get access to files**, select **Create link to files**.
+3. In the user properties page, select **OneDrive**. Under **Get access to files**, select **Create link to files**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-4.md]:
```suggestion
3. On the user properties page, select **OneDrive**. Under **Get access to files**, select **Create link to files**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**